### PR TITLE
Edits to shorten methods text

### PR DESCRIPTION
**Pull request by rando2**
### Suggested reviewers (optional)

@agitter

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by agitter** [comment]
I added the new horizontal workflow figure from https://github.com/greenelab/covid19-review/issues/1026#issuecomment-919573414 and then created a new branch on GitHub with the tracked changes from Overleaf.  I copied in the text edits and table update.  Hopefully this now covers all of the Overleaf changes and can be merged if the artifact looks okay.

I noticed the screenshot resolution in the cropped workflow figure covid-19-review-workflow-horizontal-cropped.pdf isn't as good as the original.  That's something to investigate if we have to submit another verison.

File: content/60.methods.md [hunk 1]
```diff
@@ -187,148 +181,151 @@ Manubot users can write in the simple markdown format, render the manuscript in
...
-| CI scripts to regularly download external data sources, generate figures and statistics, and store them in a GitHub branch that is read when Manubot builds the latest manuscript |
-| Zotero extension to report more relevant clinical trial metadata from <https://clinicaltrials.gov> |
-| Manubot feature to support citing any Compact Uniform Resource Identifier, such as clinicaltrials or ncbigene |
-| Integration with scite to track retractions, corrections, and notices of concern |
-| CI spell-checking reports that post as pull request comments |
-| Improved support for Pandoc's LaTeX output |
-| Building individual sections of a larger manuscript as standalone documents alongside the complete manuscript |
-
-Table: List of extensions added to Manubot's functionalities for the COVID-19 collaborative review.
+| Type | Description |
```
Comment (rando2) [inline content/60.methods.md:257]: I like the 2-column format!

**Review by rando2 — COMMENTED** [review]
This looks great to me @agitter! These are great improvements. I can't approve because I opened the PR, but I'm happy with the changes.

File: content/60.methods.md (hunk 1 above)
Comment (agitter) [reply content/60.methods.md:257]: Yeah, the 1 column table didn't look good in the LaTeX PDF.  This made it look more like an actual table and saved a little space too.

**Review by agitter — APPROVED** [review]

**Pull request merged**